docs/source/api/user-guide/combinators/compositors.rst

```diff
@@ -190,7 +192,11 @@ when the analyst may still interact with the first queryable. This is
 the subject of a further line of research on concurrent compositors,
 which we hope to make available in the next library release.
 
-An example of this constraint is demonstrated in the “Nesting” section
```

Comment:
This documentation is outdated, it's still a limitation, but only for some measures, in particular approximate zero concentrated or approximate renyi DP.

Reply:
covered by the longer provided text.